types/css.d.ts

```diff
@@ -0,0 +1,13 @@
+declare module '*.scss' {
+  const css: { [key: string]: string };
+  export default css;
+}
+declare module '*.sass' {
+  const css: { [key: string]: string };
+  export default css;
+}
+declare module 'react-markup';
```

Comment:
why this type is declared? are you using some external module with the same name?